java/tests/com/google/privacy/differentialprivacy/BoundedSumTest.java

```diff
@@ -679,4 +687,186 @@ private static void testForBias(
+                                            (Double) invocation.getArguments()[4],
+                                            (Double) invocation.getArguments()[5]));
+    sum = getBoundedSumBuilderWithFields().lower(1.0).upper(5.0).build();
+    sum.addEntry(1);
+    sum.computeResult();
+
+    assertThat(sum.computeConfidenceInterval(ALPHA))
+            .isEqualTo(ConfidenceInterval.create(0, 34.338605387474864));
+  }
+
+  @Test
+  public void computeConfidenceInterval_forLaplaceNoise() {
+    // Mock the noise mechanism. Since noise is not Laplace, nor Gaussian, delta will be passed
+    // as null, in order to pass the checks.
+    when(noise.computeConfidenceInterval(
+            anyDouble(), anyInt(), anyDouble(), anyDouble(), anyDouble(), anyDouble()))
+            .thenAnswer(
+                    invocation ->
+                            new LaplaceNoise()
+                                    .computeConfidenceInterval(
+                                            (Double) invocation.getArguments()[0],
+                                            (Integer) invocation.getArguments()[1],
+                                            (Double) invocation.getArguments()[2],
+                                            (Double) invocation.getArguments()[3],
+                                            null,
+                                            (Double) invocation.getArguments()[5]));
+    sum = getBoundedSumBuilderWithFields().lower(1.0).upper(10.0).build();
+    sum.addEntry(1);
+    sum.computeResult();
+
+    assertThat(sum.computeConfidenceInterval(ALPHA))
+            .isEqualTo(ConfidenceInterval.create(0.0, 154.0827096308998));
+  }
+
+  @Test
+  public void computeConfidenceInterval_computeResultWasNotCalled_forLong_throwsException() {
+    sum = getBoundedSumBuilderWithFields().lower(1.0).upper(5.0).build();
+    sum.addEntry(1.0);
+    IllegalStateException exception = assertThrows(IllegalStateException.class, () -> {
+      sum.computeConfidenceInterval(ALPHA);
+    });
+    assertThat(exception)
+            .hasMessageThat()
+            .startsWith("computeResult must be called before calling computeConfidenceInterval.");
+  }
+
+  @Test
+  public void computeConfidenceInterval_computeResultWasNotCalled_forDouble_throwsException() {
+    sum = getBoundedSumBuilderWithFields().lower(1.0).upper(5.0).build();
+    sum.addEntry(1.0);
+    IllegalStateException exception = assertThrows(IllegalStateException.class, () -> {
+      sum.computeConfidenceInterval(ALPHA);
+    });
+    assertThat(exception)
+            .hasMessageThat()
+            .startsWith("computeResult must be called before calling computeConfidenceInterval.");
+  }
+
+  @Test
+  public void computeConfidenceIntervals_defaultParameters_callsNoiseCorrectly() {
+    sum = getBoundedSumBuilderWithFields().lower(0.0).upper(100.0).build();
+    sum.computeResult();
+    ConfidenceInterval confInt = sum.computeConfidenceInterval(ALPHA);
```

Comment:
The local variable 'confInt' is never read. You can remove it and leave only "sum.computeConfidenceInterval(ALPHA);"

Reply:
Oops, this is like in count, missed that. Thanks!